assets/blocks/course-outline/outline-block/outline-edit.js

```diff
@@ -62,6 +69,18 @@ const OutlineEdit = ( props ) => {
 		[ clientId ]
 	);
 
+	useEffect( () => {
+		if ( ! isPostNew ) {
+			// Only add the lessons if the post is new
+			return;
+		}
+		setBlocks( [
+			{ type: 'lesson', title: __( 'Lesson 1', 'sensei-lms' ) },
+			{ type: 'lesson', title: __( 'Lesson 2', 'sensei-lms' ) },
+			{ type: 'lesson', title: __( 'Lesson 3', 'sensei-lms' ) },
```

Comment:
By the way, I added translated strings here but I'm not 100% sure if we really need to give these "lessons" a title.

With this code, when I create a new lesson I see this:

![Screenshot of the course outline block with titles in each lesson used as placeholder](https://user-images.githubusercontent.com/529864/170593361-b91c731a-01e7-4783-af13-36f39605b812.png)

Meanwhile, without the title, I see this:

![Screenshot of the course outline block but without the titles](https://user-images.githubusercontent.com/529864/170593411-425f1833-82cf-485f-af7b-9a3b7d5e8309.png)


Which IMHO doesn't look bad either.

WDYT?

Also, would you implement this translation in any different way?